CHANGELOG.md

```diff
@@ -1,16 +1,32 @@
 ## Unreleased
 
+FEATURES:
+* Support Kubernetes health probe synchronization with Consul for connect injected pods via `connectInject.healthChecks` [[GH-651](https://github.com/hashicorp/consul-helm/pull/651)].
```

Comment:
1. This needs a breaking change section as well. If any connect services were failing their kubernetes readiness checks previously then they were still routable through connect, now they won't be. Users should verify that their connect services are passing Kubernetes readiness probes prior to enabling health checks.
1. Can we add that in order to enable this feature for existing users they need to restart all the connect deployments so they are re-injected with a new label. Until they restart those deployments, health checks won't be synced.
1. I think we can be a bit more explicit here. Can we add something like "Now Kubernetes pods that fail their readiness checks will no longer receive Connect traffic until the readiness check passes"